autosklearn/ensemble_builder.py

```diff
@@ -210,17 +210,17 @@ def main(self):
             
             # train ensemble
             ensemble = self.fit_ensemble(selected_keys=selected_models)
-            
+
```

Comment:
Please revert changes in this file as there are no changes to the actual code in this file.